src/libs/mps_comm/mockup/delivery_station.cpp

```diff
@@ -35,10 +33,11 @@ MockupDeliveryStation::deliver_product(int slot)
 	assert(slot == 1 || slot == 2 || slot == 3);
 	callback_busy_(true);
 	std::lock_guard<std::mutex> lg(queue_mutex_);
-	queue_.push(
-	  std::make_tuple([this] { callback_busy_(false); },
-	                  std::chrono::system_clock::now()
-	                    + std::chrono::seconds(1 + (int)std::ceil(float(slot / exec_speed_)))));
+	queue_.push(std::make_tuple([this] { callback_busy_(false); },
+	                            std::chrono::system_clock::now()
+	                              + std::max(std::chrono::milliseconds(2000),
+	                                         std::chrono::milliseconds(
+	                                           static_cast<int>((1 + slot) / exec_speed_)))));
```

Comment:
Shouldn't this be:
```suggestion
	                                           static_cast<int>((1 + slot) * 1000 / exec_speed_)))));
```

Reply:
Ooops, true, done in e11ca9a4fd5bcaab690a1e425e2e982d9af0d1e5